README.md

```diff
@@ -113,7 +138,7 @@ var img = document.createElement('img');
 img.src = require('./glyph.png');
 ```
 
-When you require CSS (or less, etc), webpack inlines the CSS as a string inside the JS bundle and `require()` will insert a `<style>` tag into the page. When you require images, webpack inlines a URL to the image into the bundle and returns it from `require()`.
+When you require CSS (or [less](http://lesscss.org/), etc), webpack inlines the CSS as a string inside the JS bundle and `require()` will insert a `<style>` tag into the page. When you require images, webpack inlines a URL to the image into the bundle and returns it from `require()`.
```

Comment:
I'm the author of https://www.npmjs.com/package/bootstrap-sass-loader, fyi. This works well with the hot reload server.